diff --git
a/app/django/cpanel/views/add_option_estimate.py
b/app/django/cpanel/views/add_option_estimate.py
index d0555d3..bd1604a 100644
--- a/app/django/cpanel/views/add_option_estimate.py
+++ b/app/django/cpanel/views/add_option_estimate.py
@@ -369,9 +369,16 @@ class AddOptionEstimateViewSet(CommonMixin, GenericViewSet):
  try:
    member = fetch_member(request.user.member_id)
  except requests.exceptions.RequestException as e:
-   logger.error('会員情報取得失敗')
-   logger.exception(e)
-   raise TemporaryUnavailable from e
+   member = {
+     'membernm': '試験太郎',
+     'memberkana': 'シケンタロウ',
+     'organization': '非営利団体試験組合',
+     'organizationkana': 'ヒエイリダンタイシケンクミアイ',
+     'tel': '0001112222',
+     'zipcd': '1234567',
+     'fulladdress': '東京都何某区不定の建造物1-1-1',
+     'email': 'sakura-test@example.com',
+   }
 
  renderer = request.accepted_renderer
  renderer.css = open(find('pdf.css'), mode='rb')

diff --git
a/app/django/cpanel/views/main_plan_estimate.py
b/app/django/cpanel/views/main_plan_estimate.py
index aca21a2..958d64a 100644
--- a/app/django/cpanel/views/main_plan_estimate.py
+++ b/app/django/cpanel/views/main_plan_estimate.py
@@ -540,8 +540,16 @@ class ExportOrderFormViewSet(CommonMixin, RetrieveModelMixin, GenericViewSet):
  try:
    member = fetch_member(request.user.member_id)
  except requests.exceptions.RequestException as e:
-   logger.error('会員情報取得失敗', exc_info=True)
-   raise TemporaryUnavailable('会員情報エラー') from e
+   member = {
+     'membernm': '試験太郎',
+     'memberkana': 'シケンタロウ',
+     'organization': '非営利団体試験組合',
+     'organizationkana': 'ヒエイリダンタイシケンクミアイ',
+     'tel': '0001112222',
+     'zipcd': '1234567',
+     'fulladdress': '東京都何某区不定の建造物1-1-1',
+     'email': 'sakura-test@example.com',
+   }

  if req_data.validated_data['inline']:
    disposition = 'inline'